src/pages/Home.js

```diff
@@ -86,6 +86,19 @@ class Home extends Component {
       margin: '2rem auto'
     };
 
+    const buttonStyleBig = {
+      borderRadius: '5px',
```

Comment:
nit: for the stuff that's duplicated from `buttonStyle`, you can use it here as `...buttonStyle, fontSize: '22px', ...` and override the stuff to be changed directly

Reply:
great idea! thank you!